text/XXXX-access_command.md

```diff
@@ -0,0 +1,213 @@
+
+```
+$ qri access unpublish --help
+Unpublish reverses the publish process, removing a dataset from public view. 
+Unpublishing a dataset will try to "clean up", asking remotes this dataset has
+been published to
+
+Unpublished datasets cannot be pushed to the registry, but can be pushed to
+peers that accept unpublished datasets from you.
+
+Usage:
+  qri access unpublish [DATASET] [DATASET...] [flags]
+
+Examples:
+  # unpublish a dataset, removing
+  $ qri access unpublish me/dataset
+
+  # Publish a few datasets:
+  $ qri access unpublish me/dataset me/other_dataset
+ 
+Flags:
+  -r, --remove     issue "remove all" commands to any published remotes
+```
+
+
+Publish & Unpublish on their own may not seem like they warrent being moved into a subcommand. To understand the justification for the `access`, this is how `access` _might_ look one day, after we've landed encryption and role-based access control (RBAC) for datasets:
+
+```
+$ qri access --help
+Access controls visibility and permissions for datasets.
+
+Usage:
+  qri access [command]
+
+Available Commands:
+  info           show access control details for a dataset
+  publish        list dataset for others to see and pull
+  unpublish      unlist dataset, removing it from public visibility
+  encrypt        restrict access to a dataset with a secret key
+  decrypt        remove access restrictions from a dataset
+  allow          give permissions to a peer for an encrypted dataset
+  deny           remove peer access permissions
+```
+
+### `qri access info`
+```
+$ qri access info --help
+Info describes permissions for a dataset.
+
+Usage
+  qri access info [DATASET] [DATASET...]
+```
+
+An example output for a published dataset:
+```
+$ qri access info me/dataset
+can-edit:  true
+published: true
+
+You can edit this dataset.
+
+This dataset is "published". Other users may see it when you're online, and if
+pushed to the registry it will show up in feeds for other users to browse and
+pull.
+```
+
+An example for a dataset you _don't_ own:
+```
+$ qri access info ca-state-parks/park-features
+can-edit:  false
+published: true
+
+You cannot edit this dataset, attempting to save to it's history will create a 
+fork.
+
+This dataset is "published". Other users may see it when you're online, and if
+pushed to the registry it will show up in feeds for other users to browse and
+pull.
+```
+
+`$ qri access info me/dataset` for now will only but it will be _very_ helpful for viewing dataset permsissions that control both who can access a dataset and what they can do with it.
+
+### Publish on registry push
+Pushing unencrypted datasets† to the registry _requires_ a dataset be published. To keep the old behaviour of a one-liner publish, pushing to the registry will automatically run publish on the user's behalf. The push command will present this as user feedback:
+
```

Comment:
What does this imply for remotes?
Can you push a dataset to a remote but not have it published?
If it's not published and the remote only allows published datasets, is it the remote's responsibility to notify the user?